### PR TITLE
feat: claimRequireBalance fetched from contract

### PR DESCRIPTION
**Pull request by TTNguyenDev**
Issue #246 
image
image

### CODE REVIEW

**Review by aahna-ashina — CHANGES_REQUESTED** [review]
The [preview deployment](https://citizen-app-mainnet-git-feat-claimrequiredbalance-nation3dao.vercel.app/join) shows 0 instead of 3:

> image

**Comment by TTNguyenDev** [comment]
It worked on Sepolia, something is wrong on mainnet. I'm checking

**Comment by TTNguyenDev** [comment]
image
@aahna-ashina please check the site again

**Comment by TTNguyenDev** [comment]
@aahna-ashina hi please check this 
image

**Comment by TTNguyenDev** [comment]
image
This one is weird to, since we choose sepolia, but chain id = 1, can you provide me your environments?

**Comment by TTNguyenDev** [comment]
Could you provide .env file on vercel

**Comment by TTNguyenDev** [comment]
```
========= NOTICE =========
_app-eec73d473f391b30.js:1 Request-Rate Exceeded  (this message will not be repeated)
_app-eec73d473f391b30.js:1 
_app-eec73d473f391b30.js:1 The default API keys for each service are provided as a highly-throttled,
_app-eec73d473f391b30.js:1 community resource for low-traffic projects and early prototyping.
_app-eec73d473f391b30.js:1 
_app-eec73d473f391b30.js:1 While your application will continue to function, we highly recommended
_app-eec73d473f391b30.js:1 signing up for your own API keys to improve performance, increase your
_app-eec73d473f391b30.js:1 request rate/limit and enable other perks, such as metrics and advanced APIs.

```

The problem is from your Alchemy API, I believe

**Comment by TTNguyenDev** [comment]
image
@aahna-ashina some problems with user .env on vercel.
I open mainnet vercel site, but receive this message `Provider: Connected to the user's provider on chain with ID`. This message only appear if we run local node
image

**Comment by TTNguyenDev** [comment]
Use these new envs:
ALCHEMY_ID for mainnet: _lfSphd2SZQl-K3vAmW0pY9keqf9XDaB
ALCHEMY_ID for sepolia: GsbBvVaHjYdYuwkKn5_GTsMxbKvP8_QZ
@aahna-ashina

**Comment by aahna-ashina** [comment]
> Use these new envs: ALCHEMY_ID for mainnet: _lfSphd2SZQl-K3vAmW0pY9keqf9XDaB ALCHEMY_ID for sepolia: GsbBvVaHjYdYuwkKn5_GTsMxbKvP8_QZ @aahna-ashina

@TTNguyenDev Do you mean `NEXT_PUBLIC_ALCHEMY_ID`?

**Comment by TTNguyenDev** [comment]
yes

**Comment by aahna-ashina** [comment]
> yes 

@TTNguyenDev Okay, updated both ✅

Did it solve the problem?

**Comment by TTNguyenDev** [comment]
it should work, please check! @aahna-ashina

**Comment by TTNguyenDev** [comment]
@aahna-ashina Can you help me to preview this PR, it worked on my side

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (aahna-ashina) [inline yarn.lock:1]: Is this lock file needed?

**Review by aahna-ashina — APPROVED** [review]

**Pull request merged**